apiserver/keymanager/keymanager_test.go

```diff
@@ -390,7 +475,28 @@ func (s *keyManagerSuite) TestImportKeys(c *gc.C) {
 			{Error: apiservertesting.ServerError(fmt.Sprintf("duplicate ssh key: %s", key2))},
 		},
 	})
-	s.assertEnvironKeys(c, append(initialKeys, key3, keymv[0], keymv[1], keymp[0], key4))
+	s.assertKeysForModel(c, st, append(initialKeys, key3, keymv[0], keymv[1], keymp[0], key4))
+}
+
```

Comment:
please add another test that shows that users cannot touch the system key (or can touch it, if that change really was intentional)